onnxruntime/contrib_ops/cpu/quantization/dynamic_quantize_matmul.cc

```diff
@@ -114,16 +117,20 @@ Status DynamicQuantizeMatMul::Compute(OpKernelContext* ctx) const {
   const Tensor* b = packed_b_ ? nullptr : ctx->Input<Tensor>(1);
 
   const Tensor* b_scale_tensor = ctx->Input<Tensor>(2);
+#ifndef SUPPORT_COLUMNWISE_QUANTIZATION
```

Comment:
>SUPPORT_COLUMNWISE_QUANTIZATION [](start = 8, length = 31)

MLAS supports columnwise quantization. We haven't add the functionality yet. In the function ComputeCommon, you can set quant_granuity of MLAS_QGEMM_SCALE_BIAS_OUTPUT_PROCESSOR to per-column to enable the columnwise quantization.

Reply:
@yufenglee that sounds great! I guess it would come as a separate PR?

Reply:
Now, it's using MLAS per-column quantization